### DISCUSSION TITLE
你们有开放后台代码的想法没有？如果那样的话，开放度比较高，我们也比较容易融入生态。do you have an idea to open your backend code ? if do it ,the open communti is very higher!

### DISCUSSION THREAD

**Discussion by lishu2006ll** [opentiny/tiny-engine#173] [General]
你们有开放后台代码的想法没有？如果那样的话，开放度比较高，我们也比较容易融入生态。 Do you have any plans to open-source your backend code? If so, it would significantly increase community participation!

**Comment by chilingling** [comment]
请参考场景问题答疑：https://github.com/opentiny/tiny-engine/issues/78
里面有提到后台代码会在本月底左右的时间节点开源

**Comment by wenmine** [comment]
后端代码已经开源~具体请看：https://github.com/opentiny/tiny-engine-data-center 和 https://github.com/opentiny/tiny-engine-webservice
